src/style.css

```diff
@@ -0,0 +1,154 @@
+body {
```

Comment:
There's a CSS file under `src/css/style.css`, please add these styles to that file

Also, please use class selector, you might wanna read about BEM convention

Reply:
We're not strict BEM convention, just use class selectors 😄